client/packages/react-ui/src/types/svg.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module '*.svg' {
```

Comment:
For what is this?

Reply:
We need this module declaration to support import statements for '.svg' files.
This may sometimes also be provided by a build system, e.g., see type declarations in CRA: https://github.com/facebook/create-react-app/blob/main/packages/react-scripts/lib/react-app.d.ts#L47 

I will add a comment to the file to clarify this.